.github/workflows/PullRequestClosed.yml

```diff
@@ -14,7 +14,6 @@ jobs:
     # For external PR, ticket should be moved manually
     if: |
         github.event.pull_request.head.repo.full_name == github.repository
-        && github.event.pull_request.merged
```

Comment:
This line being removed, it means the `PullRequestClosed@v2` action will trigger even if the pull request is closed without being merged? If it's the case then you could also update the job name that is now imprecise:


```
jobs:
  PullRequestMerged_job:
    name: Pull Request Merged
```